src/components/RunTable/RunRow.jsx

```diff
@@ -15,14 +15,13 @@ const RunRow = ({ runs, run, locateActivity, runIndex, setRunIndex }) => {
 
   // change click color
   const handleClick = (e, runs, run) => {
-    const elementIndex = runs.indexOf(run);
-    e.target.parentElement.style.color = 'red';
-
     const elements = document.getElementsByClassName(styles.runRow);
-    if (runIndex !== -1 && elementIndex !== runIndex) {
-      elements[runIndex].style.color = MAIN_COLOR;
+    for (let i = 0; i < elements.length; i += 1) {
+      elements[i].style.color = MAIN_COLOR
     }
-    setRunIndex(elementIndex);
+    const elementIndex = runs.indexOf(run)
+    e.target.parentElement.style.color = 'red'
+    setRunIndex(elementIndex)
   };
```

Comment:
do we have some better solution
this need to  loop the elements every time?

Reply:
My girlfriend said yes, but she couldn't tell me until the weekend ~

Reply:
I figure out some better solution, will push use your `PR`